packages/components/src/navigator/stories/index.tsx

```diff
@@ -0,0 +1,210 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../button';
+import { Card, CardBody, CardFooter, CardHeader } from '../../card';
+import { HStack } from '../../h-stack';
+import Dropdown from '../../dropdown';
+import {
+	NavigatorProvider,
+	NavigatorScreen,
+	NavigatorButton,
+	NavigatorBackButton,
+} from '..';
+
+const meta: ComponentMeta< typeof NavigatorProvider > = {
+	component: NavigatorProvider,
+	title: 'Components (Experimental)/Navigator',
+	subcomponents: { NavigatorScreen, NavigatorButton, NavigatorBackButton },
+	argTypes: {
+		as: { control: { type: null } },
+		children: { control: { type: null } },
+		initialPath: { control: { type: null } },
```

Comment:
I disabled the control for `initialPath` because changing it doesn't make any difference (it's only used on the component's first render)